authentication/encrypting-etcd.adoc

```diff
@@ -0,0 +1,15 @@
+[id="encrypting-etcd"]
+= Encrypting etcd data
+include::modules/common-attributes.adoc[]
+:context: encrypting-etcd
+
+toc::[]
+
+// About etcd encryption
+include::modules/about-etcd-encryption.adoc[leveloffset=+1]
+
+// Enabling etcd encryption
+include::modules/enabling-etcd-encryption.adoc[leveloffset=+1]
+
+// Disabling etcd encryption
```

Comment:
I think that the disabling module should go in a separate assembly because they're not part of the same user goal.